examples/net_client/main.rs

```diff
@@ -0,0 +1,48 @@
+extern crate amethyst;
+
+use std::time::Duration;
+
+use amethyst::Result;
+use amethyst::audio::AudioBundle;
+use amethyst::core::frame_limiter::FrameRateLimitStrategy;
+use amethyst::core::transform::TransformBundle;
+use amethyst::ecs::{Component, DenseVecStorage};
+use amethyst::input::InputBundle;
+use amethyst::prelude::*;
+use amethyst::renderer::{DisplayConfig, DrawFlat, Pipeline, PosTex, RenderBundle, RenderSystem,
+                         Stage};
+use amethyst::ui::{DrawUi, UiBundle};
+use amethyst::network::network_client::*;
+use amethyst::network::resources::net_event::*;
+use amethyst::shrev::EventChannel;
+use std::net::IpAddr;
+use std::net::SocketAddr;
+use std::str::FromStr;
+
+fn main() {
+    if let Err(e) = run() {
+        println!("Failed to execute example: {}", e);
```

Comment:
log

Reply:
you know I can Ctrl+f  "println!" right? lol

Reply:
:P Partially done just to be silly but also partially done in case whatever find tool you have misses something.  I know I've missed things with "find all" tools before.

Reply:
okay ;)